src/marvin/beta/assistants/formatting.py

```diff
@@ -151,7 +151,7 @@ def pprint_message(message: ThreadMessage):
     }
 
     color = role_colors.get(message.role, "red")
-    timestamp = datetime.fromtimestamp(message.created_at).strftime("%l:%M:%S %p")
+    timestamp = datetime.fromtimestamp(message.created_at).strftime("%I:%M:%S %p")
```

Comment:
lowercase l will strip the leading 0 zero, which we can still do because it looks nice 🙂 
```suggestion
    timestamp = (
        datetime.fromtimestamp(message.created_at).strftime("%I:%M:%S %p").lstrip("0")
    )
```